uid-pages/layout-bonita/src/LayoutBonitaV6.json

```diff
@@ -2594,8 +2594,15 @@
         "if($data.eventList.actions && $data.eventList.actions.length > 0){",
         "    var action=$data.eventList.actions[0];",
         "    if(action===\"locale\"){",
-        "        document.cookie=\"BOS_Locale=\"+$data.selectedLanguage.locale+\";path=/\";;",
-        "        document.location.reload();",
+        "        document.cookie=\"BOS_Locale=\"+$data.selectedLanguage.locale+\";path=/\";",
+        "        try {",
+        "            var url = new URL(document.location.href);",
```

Comment:
maybe a test would be nice for this. Navigate to the url with ?locale and _l and check that it doesn't contain it anymore when changing the language.

Reply:
I can do it if you don't have the time.

Reply:
I will check if it is possible with cypress

Reply:
done